libs/ui/src/lib/treemap-chart/treemap-chart.component.ts

```diff
@@ -47,6 +49,9 @@ export class GfTreemapChartComponent
   @Input() cursor: string;
   @Input() dateRange: DateRange;
   @Input() holdings: PortfolioPosition[];
+  @Input() colorScheme: ColorScheme;
+  @Input() baseCurrency: string;
```

Comment:
Can you move up ⬆️  to keep the alphabetical order?

Reply:
Done